apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Getting started
+description: Getting started with Workleap Design Tokens
+---
+
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/38871812/186cfadb-8c57-4994-ad97-5030a960700a)

weird looking in dark mode